FutureCollars kurs/src/main/java/com/futurecollars/lesson5/task1/TextProcessor.java

```diff
@@ -0,0 +1,17 @@
+package com.futurecollars.lesson5.task1;
+
+
+public class TextProcessor {
+    public static void main(String[] args) {
+        String text = "rotator";
+        boolean isPalindrome = PalindromeChecker.isPalindrome(text);
```

Comment:
Ta logika jest nam dopiero potrzebna w przypadku "else". Jeśli umieścisz ją tutaj to metody isPalindrome oraz getTextLength zawsze zostaną wywołane, a nie zawsze jest nam to potrzebne.